test/integration/kubernetes_deploy_test.rb

```diff
@@ -11,7 +11,7 @@ def test_full_hello_cloud_set_deploy_succeeds
       "Deploying ConfigMap/hello-cloud-configmap-data (timeout: 30s)",
       "Hello from the command runner!", # unmanaged pod logs
       "Result: SUCCESS",
-      "Successfully deployed 14 resources"
+      "Successfully deployed 15 resources"
     ], in_order: true)
 
     assert_logs_match_all([
```

Comment:
Can you please add an assertion for the logs about your new fixture here? (ie. `%r{Service/redis-external\s+Doesn't require any endpoint}`)